CHANGELOG.md

```diff
@@ -35,6 +35,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Removed
     - Issue 18 - Remove Flask
     - Issue 52 - Remove partial_f from data columns to obscure
+    - Issue 39 - Clean up code, removed comments and unused parameters
```

Comment:
All changelog entries getting merged to `develop` should be added to the `Unreleased` section

Reply:
done